carla/models/catalog/catalog.py

```diff
@@ -41,8 +43,10 @@ def __init__(
             The directory in which to cache data; see :func:`get_models_home`.
         kws : keys and values, optional
             Additional keyword arguments are passed to passed through to the read model function
-        data : data.api.Data Class
-            Correct dataset for ML model
+        encode_normalize_data : bool, optional
+            If true, the model pipeline is used to build data.encoded, data.normalized and data.encoded_normalizd
```

Comment:
Related about the comment wether we should still keep the encoded/normlized information in the data catalog. Which usecases do you forsee with that?
Here (as mentioned above) we are tightly coupling these two components together. If the user really needs to access scaled/encoded data outside of the MLModel it could instead scale it themselves by accessing the MLModels scaler?

Reply:
Until now, it is only used as a quick way to access scaled and encoded data, without calling scaler/ encoder.
As mentioned in the comment above, we could get rid of that